app/src/main/java/com/kevinmost/overlauncher/app/App.java

```diff
@@ -7,8 +7,9 @@
 
 public class App extends Application {
   private static App INSTANCE;
-
+  
   private final AppModule module = new AppModule(this);
+  private final AppComponent appComponentInstance = DaggerAppComponent.builder().appModule(INSTANCE.module).build();
```

Comment:
Replace `INSTANCE.module` with just `module`. You're already within the same singleton instance, no need to specify that.